src/main/java/co/rsk/bitcoinj/script/P2shErpRedeemScriptParser.java

```diff
@@ -1,28 +1,61 @@
 package co.rsk.bitcoinj.script;
 
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.Sha256Hash;
 import co.rsk.bitcoinj.core.Utils;
 import co.rsk.bitcoinj.core.VerificationException;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public class P2shErpFederationRedeemScriptParser extends StandardRedeemScriptParser {
-    private static final Logger logger = LoggerFactory.getLogger(P2shErpFederationRedeemScriptParser.class);
+public class P2shErpRedeemScriptParser implements RedeemScriptParser {
+    private static final Logger logger = LoggerFactory.getLogger(P2shErpRedeemScriptParser.class);
 
     public static long MAX_CSV_VALUE = 65_535L; // 2^16 - 1, since bitcoin will interpret up to 16 bits as the CSV value
 
-    public P2shErpFederationRedeemScriptParser(
+    private final RedeemScriptParser internalStandardMultiSigRedeemScriptParser;
```

Comment:
Didn't we agree on calling this the `default` redeem script?

Reply:
Updated